internal/querycoord/meta.go

```diff
@@ -99,6 +104,7 @@ type MetaReplica struct {
 	segmentMu         sync.RWMutex
 	queryChannelInfos map[UniqueID]*querypb.QueryChannelInfo
 	channelMu         sync.RWMutex
+	deltaChannelInfos sync.Map
```

Comment:
I think it might be better to add a deltaChannelInfo to proto querypb.collectionInfo
```
message CollectionInfo {
  int64 collectionID = 1;
  repeated int64 partitionIDs = 2;
  repeated PartitionStates partition_states = 3;
  repeated DmChannelInfo channel_infos = 4;
  LoadType load_type = 5;
  schema.CollectionSchema schema = 6;
  repeated int64 released_partitionIDs = 7;
  int64 inMemory_percentage = 8;
}
```